### PR TITLE
Disable gcs azure daily test

### PR DESCRIPTION
**Pull request by bigning**
For GCS key, we're waiting for the request https://databricks.atlassian.net/browse/SOPS-719?focusedCommentId=5299281

For Azure, @j316chuck  will start the request once we go through the GCS request process. 

# **before cpu test failure:**
https://github.com/mosaicml/composer/actions/runs/10207147591
image

# **after cpu tests succeed:**
[gpu tests still running] https://github.com/mosaicml/composer/actions/runs/10207236494
image

### CODE REVIEW

File: tests/utils/object_store/test_azure_object_store.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 
 @pytest.mark.remote
+@pytest.mark.skip(reason='Waiting new Azure key to be approved')
```
Comment (snarayan21) [inline tests/utils/object_store/test_azure_object_store.py:12]:
```suggestion
@pytest.mark.skip(reason='Waiting for new Azure key to be approved')
```

File: tests/utils/object_store/test_gs_object_store.py [hunk 2]
```diff
@@ -25,6 +25,7 @@ def get_gcs_os_from_trainer(trainer: Trainer) -> GCSObjectStore:
 
 @pytest.mark.gpu  # json auth is hard to set up on github actions / CPU tests
 @pytest.mark.remote
+@pytest.mark.skip(reason='Waiting new GCP key to be approved')
```
Comment (snarayan21) [inline tests/utils/object_store/test_gs_object_store.py:28]:
```suggestion
@pytest.mark.skip(reason='Waiting for new GCP key to be approved')
```

File: tests/utils/object_store/test_gs_object_store.py [hunk 3]
```diff
@@ -63,6 +64,7 @@ def test_gs_object_store_integration_hmac_auth(expected_use_gcs_sdk_val=False, c
 
 @pytest.mark.gpu
 @pytest.mark.remote
+@pytest.mark.skip(reason='Waiting new GCP key to be approved')
```
Comment (snarayan21) [inline tests/utils/object_store/test_gs_object_store.py:67]:
```suggestion
@pytest.mark.skip(reason='Waiting for new GCP key to be approved')
```

**Review by snarayan21 — APPROVED** [review]
lgtm, minor comments + quick stamp

**Review by j316chuck — APPROVED** [review]

**Review by mvpatel2000 — APPROVED** [review]

**Pull request merged**